### PR TITLE
Update Edge install instructions

### PR DESCRIPTION
**Pull request by emdobrin**

### CODE REVIEW

File: foundation-extensions/experience-platform-extension/README.md [hunk 1]
```diff
@@ -31,82 +38,85 @@ The Edge configuration used by the client-side implementation is one of the foll
...
-@import AEPIdentity;
-@import AEPLifecycle;
-@import AEPSignal;
+@import AEPEdgeIdentity;
 ```
 {% endtab %}
+
+{% tab title="iOS (ACP 2.x)" %}
+
+This extension is built based on the AEPCore, please follow [the guide for migrating to Swift](https://aep-sdks.gitbook.io/docs/resources/migrate-to-swift).
```
Comment (kevinlind) [inline foundation-extensions/experience-platform-extension/README.md:95]: Maybe reword to:
"This extension is built on the AEPCore (3.x) and cannot be used with the ACPCore (2.x). Please follow the guide for migrating to the Swift AEPCore."

**Review by kevinlind — APPROVED** [review]
I added a comment about rewording a sentence, but otherwise looks good.

**Review by cacheung — APPROVED** [review]
